docker-compose.yml

```diff
@@ -28,6 +28,8 @@ services:
        - "./wordpress:/var/www/html"
        - ".:/root"
        - "./site.conf:/etc/apache2/sites-available/000-default.conf"
+     extra_hosts:
+       - "apps.memberful.localhost:host-gateway"
```

Comment:
Running `docker exec memberful-wp_wordpress_1 cat /etc/hosts` after building with this option shows this added in the hosts file:

`192.168.65.1	apps.memberful.localhost`

But trying to curl that domain results in nothing. For comparison, a curl to `host.docker.internal` from inside the container gives me the "unknown app" response I would expect from post 80 of my host machine.